src/about.html

```diff
@@ -6,16 +6,16 @@
 
 <h1>About</h1>
 <p>
-  <span class="bold">Neurodiversity In Tech</span> is an online community dedicated to supporting
+  <span class="bold">Neurodiversity In Tech</span> is an online Slack community dedicated to supporting
   neurodivergent professionals in the technology industry.
 </p>
 <p>
-  The dynamic Slack Workspace enables members to network, navigate their careers, and empower each other while
+  Our dynamic network enables members to network, navigate their careers, and empower each other while
```

Comment:
Thanks! I like the idea of community in both or switching which we use where. Which do you think best of those 2 options?

Reply:
I quite like "network" for the second one in this sentence. but either way it needs to be a verb - I think you've made a very nice space for networking, but I'm not sure it would work to be communitying :grin: 

But I suspect I've missed something, or maybe not communicated the idea well in the first place. Which other use of network were you thinking we could change?

Reply:
Fixed!